src/ui/icons.ts

```diff
@@ -35,3 +35,10 @@ export const documentIcon =
   '<svg xmlns="http://www.w3.org/2000/svg" version="1.0" width="220" height="220" viewBox="0 0 440 440">' +
   '  <path d="M 390,220 V 440 H 215 40 V 278 116 L 97,58 154,0 H 272 390 Z M 124,130 H 80 V 265 400 H 215 350 V 220 41 H 170 v 90 0 z"/>' +
   '</svg>';
+
+  export const upLevelIcon = 
+  `<svg xmlns="http://www.w3.org/2000/svg" version="1.0" width="220" height="220" viewBox="0 0 200 200" preserveAspectRatio="xMidYMid meet">
+    <g transform="translate(0.000000,200.000000) scale(0.100000,-0.100000)" stroke="none">
+    <path d="M120 1000 l0 -780 880 0 880 0 -2 662 -3 663 -575 3 -575 2 -115 115 -115 115 -187 0 -188 0 0 -780z m908 186 c15 -8 69 -57 120 -109 98 -100 111 -130 75 -169 -27 -30 -60 -22 -113 28 l-50 46 -2 -193 -3 -193 -27 -16 c-25 -13 -31 -13 -55 0 l-28 16 -3 193 -2 193 -50 -46 c-28 -26 -59 -46 -71 -46 -32 0 -59 29 -59 63 0 24 18 48 98 129 81 84 115 112 140 117 2 1 15 -5 30 -13z"/>
+    </g>
+  </svg>`;
```

Comment:
In no way a blocker, but I always like to pass SVG markup through this fun little clean up tool https://jakearchibald.github.io/svgomg/

Reply:
Great suggestion! I just ran it through and pushed the cleaned up version.